### ISSUE TITLE
System->Alarm->Sound Off

### ISSUE THREAD

**Issue by jmxp69** [EdgeTX/edgetx#2348]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

2.8 self build on 9/22/22. Two issues:

1) Minor Issue: Radio Setup the button is labeled Alarm. When pressing it the title is Alarms (plural). Probably should be consistent.

2) Main Issue: In Alarm(s), with sound off toggled, it does not prevent verbal alerts for either the battery falling below the set threshold or inactivity. When either of those conditions are violated, the radio audio prompts, therefore; it is not intuitive what sound the sound off feature is supposed to suppress.

### Expected Behavior

I would expect sound off to disable audio for both alerts on this page. It probably should be further refined as:

Battery Low (value)
Sound off

Inactivity (value)
Sound off

### Steps To Reproduce

1. System-Setup
2. Alarm(s)
3. Sound Off = Toggled affirmative.
4. Set Battery low threshold to something higher than your current battery level. i.e. 8.9
5. Note audible alarm.
6. Set Inactivity to 1 minute.
7. Wait 1 minute.
8. Note audible alarm.

### Version

Nightly (Please give date/commit below)

### Transmitter

Radiomaster TX16S

### Anything else?

2.8.0 Self Build on 9/22/22.

**Comment by pfeerick** [comment]
no. 2 could be clearer... As it's not sound off as in turn sound off - it is the alarm that the sound *IS* turned off... i.e. leave that enabled, to into sounds, and change your actual sound mode to Quiet. Turn the radio off, and on again... you'll get a warning that the sound is off. If you go back into Alarms, and disable that "sound off" alarm, and cycle the power again, you shouldn't get the "no sound" warning this time.
